doc: de-duplicate instructions in kind and hubble getting started guides (#12210)
cilium-team/cilium: Documentation/gettingstarted/hubble.rst

   This terminal window needs to be remain open to keep port-forwarding in
   place. Open a separate terminal window to use the ``hubble`` CLI.

You may verify that your setup is working by issuing this command which gives

> you the status of the Hubble Relay service:

Reviewer: nit:

```suggestion
Confirm that the Hubble Relay service is healthy via `hubble status`:
```
